src/System.Runtime/tests/System/IntPtr.cs

```diff
@@ -3,131 +3,192 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-
+using System.Collections.Generic;
 using Xunit;
 
-public static unsafe class IntPtrTests
+public static class IntPtrTests
 {
+    private static unsafe bool s_is32Bits = sizeof(void*) == 4; // Skip IntPtr tests on 32-bit platforms
```

Comment:
I realize this is a hold-over from the existing tests, but it's not clear to my why we skip all such testing on 32-bit.  It'd be good to open an issue to track this.  Then until that's addressed, you can use ```[ConditionalFact(nameof(Is64Bit)]``` with a property like:
```C#
private static unsafe bool Is64Bit => sizeof(void*) == 4;
```
rather than using ```[Fact]``` and having to check ```s_is32Bits``` at the beginning of each method.